Examples/Tests/collision/inputs_1d

```diff
@@ -0,0 +1,93 @@
+#################################
+########## CONSTANTS ############
+#################################
+
+my_constants.epsilon = 1.0e-12
```

Comment:
That is there so that one can distinguish the two population of particles in the output file for test T1a when using equally-weighted particles. I didn't know how else to do it other than having one population with a slightly different weight than the other.

Reply:
Since the CI test does not do test T1a. I can remove the epsilon.